services/core-web/src/styles/settings/themeOverride.scss

```diff
@@ -551,6 +551,10 @@
   padding: 0;
 }
 
+.ant-modal-footer .ant-btn {
+  margin: 10px;
```

Comment:
A note for the future:

Writing it nested like:
`.ant-modal-footer { .ant-btn { margin: 10px }}`
did produce a build error in the javascript bundle. In order to reproduce the bundle error, just change it back to nested.

However, leaving it in, and adding `.ant-modal-footer .ant-btn {...` did not produce a build error. So it is not the presence or absence of that particular rule/format that breaks the build.